src/CMakeLists.txt

```diff
@@ -18,6 +18,12 @@ list(APPEND SRCS args.cc baseclient.cc client.cc http.cc request.cc response.cc
 add_library(miniocpp STATIC ${SRCS})
 target_link_libraries(miniocpp ${requiredlibs})
 
+if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU"
+    AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 8.0
+    AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0 AND NOT MINGW)
+    target_compile_options(miniocpp PUBLIC -lstdc++fs)
```

Comment:
This should be 

```
target_link_libraries(miniocpp stdc++fs)
```

Reply:
Yep, just thought target_ compile_options had same effect